code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
+        <h2>Broccoli 🥦</h2>
+        <p>A humble and delicious vegetable</p>
+        <p>Here are my favourite things about it</p>
```

Comment:
Should we add a colon? A colon would be nice since it indicates the start of a list.